### PR TITLE
Create .githuFUNDING.ymlioy

### PR DESCRIPTION
**Pull request by Abkdtrr**
### Why:

Closes: 

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by Abkdtrr** [comment]
Try

**Comment by Abkdtrr** [comment]
Try

File: .githuFUNDING.yml
Comment (ramona013) [inline .githuFUNDING.yml:1]: Jammer joh. Dit geeft mensen gratis een kans om een basis te ebben om te starten ipv voor bakken met geld

**Review by Blonde488 — COMMENTED** [review]
Great app!

**Pull request closed without merging**